tests/e2e/configurer/upgrade.go

```diff
@@ -127,6 +128,16 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	chainANode.CreateBalancerPool("pool1A.json", initialization.ValidatorWalletName)
 	chainBNode.CreateBalancerPool("pool1B.json", initialization.ValidatorWalletName)
+	if uc.IsEnabled {
+		chainA.SendIBC(chainB, chainB.NodeConfigs[0].PublicAddress, initialization.StableTokenA)
+		chainB.SendIBC(chainA, chainA.NodeConfigs[0].PublicAddress, initialization.StableTokenA)
+		chainA.SendIBC(chainB, chainB.NodeConfigs[0].PublicAddress, initialization.StableTokenB)
+		chainB.SendIBC(chainA, chainA.NodeConfigs[0].PublicAddress, initialization.StableTokenB)
+		chainANode.CreateStableSwapPool("stablePool.json", initialization.ValidatorWalletName)
+		chainBNode.CreateStableSwapPool("stablePool.json", initialization.ValidatorWalletName)
+		chainANode.CreateConcentratedPool(initialization.ValidatorWalletName, "uion", "uosmo", 1, -1, "0.01")
+		chainBNode.CreateConcentratedPool(initialization.ValidatorWalletName, "uion", "uosmo", 1, -1, "0.01")
```

Comment:
Actually, I think we should hold-off with pre-creating CL pools until after concentrated liquidity is released. The reason is there are no CL pools existing in the pre-upgrade version